scripts/datageneration/retrieve_combinations.py

```diff
@@ -144,8 +144,15 @@ def process_tag_attributes(self, tag_df):
                         tag_operator = seperator
                         continue
 
-                all_osm_tags_and_attributes[_tag] = {'tag': _tag, 'key': _tag_splits[0], 'operator': tag_operator,
+                if _tag in all_osm_tags_and_attributes:
+                    if all_osm_tags_and_attributes[_tag]["type"] != tag_type:
```

Comment:
This ensures that all tags that should be tagged as "core/attr" are indeed tagged as such.
Before, if a tag appeared in multiple bundles, only the type of the last occurrence was used as the final type.

This method checks whether a given tag occurs for the second time, and whether the new type is the same as before (otherwise is uses "core/attr", as this is the logical consequence if two different types were found for this tag).